bark/benchmark/benchmark_result.py

```diff
@@ -0,0 +1,289 @@
+# Copyright (c) 2019 fortiss GmbH
+#
+# This software is released under the MIT License.
+# https://opensource.org/licenses/MIT
+
+import os
+import pickle
+import pandas as pd
+import logging
+import re
+import zipfile
+import math
+
+logging.basicConfig()
+logging.getLogger().setLevel(logging.INFO)
+
+
+class BehaviorConfig:
```

Comment:
maybe add some documentation to especially the scenario generation.
It is not as easy to grasp.